code/controllers/configuration/entries/game_options.dm

```diff
@@ -107,9 +107,24 @@
 
 /datum/config_entry/flag/protect_assistant_from_antagonist //If assistants can be traitor/cult/other
 
-/datum/config_entry/flag/enforce_human_authority //If non-human species are barred from joining as a head of staff
+/datum/config_entry/number/human_authority //Controls how to enforce human authority
+	default = 0
+	min_val = 0
+	max_val = 3
+
+/////////////////////////////////////////////////Outdated human authority settings
+/datum/config_entry/flag/enforce_human_authority
+	deprecated_by = /datum/config_entry/number/human_authority
 
-/datum/config_entry/flag/enforce_human_authority_on_everyone //If non-human species are barred from joining as a head of staff, including jobs flagged as allowed for non-humans, ie. Quartermaster.
+/datum/config_entry/flag/enforce_human_authority/DeprecationUpdate(value)
+	return 2
```

Comment:
oh wait, technically you would return 2 unless value is 0, which you would return 0

same for the other one. but tbh unsetting these configs is hardly supported given ambiguities with the two of them so this is still fine.

Reply:
Nooot entirely sure what you mean here. Since it's a flag, the only way it could be 0 is if it's commented out, and if it's commented out then the deprecated config system won't kick in, since it can't see it, so it can never be 0

Reply:
all flags can be set to 0 to turn them off in later config $includes. (ie turning something off per-server thats enabled in a global config file for all servers, like at /tg/)

Reply:
Ooooh, I'll fix it in a bit then